Wikipedia/Code/UIColor+WMFHexColor.h

```diff
@@ -5,6 +5,6 @@
 + (UIColor *)wmf_colorWithHex:(NSInteger)hex
                         alpha:(CGFloat)alpha;
 
-- (NSString *)wmf_hexString;
+- (NSString *)wmf_hexStringWithAlpha:(BOOL)useAlpha;
```

Comment:
Two methods would be better than one here. Instead of requiring that alpha parameter to be passed on every use.

Also, the naming could be a little better, I wasn't totally clear what would happen until I looked at the implementation.

`wmf_hexStringIncludingAlpha:` seems more clear.

Also, since this is returning a string - it would be nice to have a comment in the header file showing the format of the output.